DomainContent/Rust/DJ-Tools/DJ-Tools_Spawner.js

```diff
@@ -0,0 +1,560 @@
+        GROUP_RIGHT = "Group_Right",
+        DEBUG_CUBE = "debugCube",      
+        INPUT = "input",
+        ENDPOINT = "endPoint";
+
+    // Polyfill
+    Script.require(Script.resolvePath("./Polyfills.js"))();
+
+    // Colections
+    var phlashProps = gProps(PHLASH_TABLE_NAME),
+        allEnts = [],
+        particleBaseProps = {
+            "type": "ParticleEffect",
+            "isEmitting": true,
+            "lifespan": "2.0299999713897705",
+            "maxParticles": "6717",
+            "textures": "http://hifi-content.s3.amazonaws.com/alan/dev/Particles/Bokeh-Particle.png",
+            "emitRate": "0",
+            "emitSpeed": "1.47",
+            "emitDimensions": {
+                "x": "0.5",
+                "y": "0.5",
+                "z": "0.5"
+            },
+            "emitOrientation": {
+                "x": "-90.01",
+                "y": "0",
+                "z": "0"
+            },
+            "emitterShouldTrail": true,
+            "particleRadius": "0",
+            "radiusSpread": "4",
+            "radiusStart": "0.5799999833106995",
+            "radiusFinish": "0",
+            "color": {
+                "red": "255",
+                "blue": "255",
+                "green": "255"
+            },
+            "colorSpread": {
+                "red": "0",
+                "blue": "0",
+                "green": "0"
+            },
+            "colorStart": {
+                "red": "255",
+                "blue": "33",
+                "green": "33"
+            },
+            "colorFinish": {
+                "red": "239",
+                "blue": "255",
+                "green": "13"
+            },
+            "emitAcceleration": {
+                "x": "0.01",
+                "y": "0.01",
+                "z": "0.01"
+            },
+            "accelerationSpread": {
+                "x": "1",
+                "y": "1",
+                "z": "1"
+            },
+            "alpha": "0.6000000238418579",
+            "alphaSpread": "0",
+            "alphaStart": "0.09000000357627869",
+            "alphaFinish": "0",
+            "polarStart": "0",
+            "polarFinish": "1.02974",
+            "azimuthStart": "-180.00000500895632",
+            "azimuthFinish": "180.00000500895632"
+        },
+        lightBaseProps = {
+            type: "Light",
+            angularDamping: 0,
+            color: {
+                red: 255,
+                blue: 255,
+                green: 255
+            },
+            intensity: 0,
+            falloffRadius: 0,
+            isSpotlight: 0,
+            exponent: 1,
+            cutoff: 10,
+            collisionless: true
+        },
+        barrelStageLeftPosition = {
+            x: -26.4579,
+            y: 4989.7456,
+            z: -23.3428
+        },
+        barrelStageRightPosition = {
+            x: -38.6957,
+            y: 4989.5752,
+            z: -23.3843
+        },
+        entityNames = [
```

Comment:
Do these need to be referencing a specific DJ?

Reply:
Changed manually inputting to auto generated, and stored in settings if it's the first time(this is just to make it easier to delete if I crash in dev)